bindings/python/stp/stp.py.in

```diff
@@ -606,7 +606,8 @@ class Expr(object):
     @property
     def value(self):
         """Returns the value of this BitVec in the current model."""
-        return self.s.model(self.name)
+        value = _lib.vc_getCounterExample(self.s.vc, self.expr)
```

Comment:
How about changing `Solver.model()` to accept either a name *or* an Expr instance?